src/SDKs/HDInsight/Management.HDInsight/Management.HDInsight.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.HDInsight</PackageId>
+    <Description>Azure HDInsight Management SDK Library</Description>
+    <AssemblyName>Microsoft.Azure.Management.HDInsight</AssemblyName>
+    <VersionPrefix>3.1.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure HDInsight Management;HDInsight;HDInsight Management</PackageTags>    
```

Comment:
@wawon-msft missing PackageReleaseNotes, see e.g.
https://github.com/Azure/azure-sdk-for-net/blob/psSdkJson6/src/SDKs/BatchAI/Management.BatchAI/Microsoft.Azure.Management.BatchAI.csproj#L10

Reply:
Resolved in PR #3845.